yarn-project/p2p/src/services/libp2p/libp2p_service.ts

```diff
@@ -144,95 +143,6 @@ export class LibP2PService<T extends P2PClientType> extends WithTracer implement
     };
   }
 
-  /**
```

Comment:
I just moved start and stop below the static initialiser just so that it was closer to the constructor